src/main/java/org/jabref/logic/importer/fileformat/CitaviXmlImporter.java

```diff
@@ -188,7 +189,12 @@ private BibEntry parseData(CitaviExchangeData.References.Reference data) {
                 .ifPresent(value -> entry.setField(StandardField.DOI, clean(value)));
         Optional.ofNullable(data.getIsbn())
                 .ifPresent(value -> entry.setField(StandardField.ISBN, clean(value)));
-        entry.setField(StandardField.PAGES, clean(getPages(data)));
+
+        String pages = clean(getPages(data));
+        // Cleans also unicode minus signs
+        pages = new NormalizePagesFormatter().format(pages);
```

Comment:
I think, the lifecycle of our formatters is longer --> introduce the formatter as class variable?

Reply:
Move this up now as a field